### DISCUSSION TITLE
Slack Notifications:  Does the documentation need an update?

### DISCUSSION THREAD

**Discussion by loujacob66** [cerb/cerb-release#85] [Q&A]
Hello -

I'm working through the Slack integrations and am stuck on Slack Notifications, which are documented here:

https://cerb.ai/workflows/cerb.integrations.slack.notifications/

Per the instructions, I should be able to add custom fieldsets to configure the workflow, however there is no "add fieldset" button on any of the groups where I can select the Slack application.

I've done the initial integration as documented, using Bot Tokens, documented here:

https://cerb.ai/guides/integrations/slack/

Have I missed a different step which will surface the "add fieldset" option, or is a different approach required?

Thanks,
Lou

<img width="1159" alt="image" src="https://github.com/user-attachments/assets/65d6b07e-12ed-4fb6-aa0f-1fce9be83da8" />

**Comment by loujacob66** [comment]
Reinstalled and went through the configuraton of the notifications workflow again and this time it appears to have worked.

**Reply by jstanden** [reply]
Strange! When you view a workflow it'll show you the records it manages. I'd be curious if the custom fieldset was listed there.

Let me know if it happens again.
